R/dfm_select.R

```diff
@@ -70,37 +64,32 @@
 #' txts <- c("This is text one", "The second text", "This is text three")
 #' (dfm1 <- dfm(txts[1:2]))
 #' (dfm2 <- dfm(txts[2:3]))
-#' (dfm3 <- dfm_select(dfm1, dfm2, valuetype = "fixed", padding = TRUE, verbose = TRUE))
+#' (dfm3 <- dfm_select(dfm1, dfm2, valuetype = "fixed", verbose = TRUE))
 #' setequal(featnames(dfm2), featnames(dfm3))
 #' 
-dfm_select <- function(x, pattern = NULL, documents = NULL, 
```

Comment:
As you will see, I killed the document selection from `dfm_select()`. If we want to implement document selection, I suggest `dfm_subset()` as described in https://github.com/kbenoit/quanteda/issues/854#issuecomment-315523396.